### ISSUE TITLE
GMail connection error

### ISSUE THREAD

**Issue by jonas-kgomo** [hcengineering/platform#5809]
When i try to connect Google Gmail in my intergrations it gives this error
![Screenshot 2024-06-13 at 16 22 12](https://github.com/hcengineering/platform/assets/25654848/5635b0d6-e711-40bc-ac15-36a74a5439ba)

**Comment by BykhovDenis** [comment]
Yes, we still waiting verification from google, currently it's in the test mode.
